packages/net-modules.spec

```diff
@@ -32,13 +29,15 @@ The first implementation in this repository showcases Apache Mesos using Project
 %build
 ./bootstrap
 
-%configure --with-mesos=/usr/include/mesos/ --with-protobuf=/usr
+CPPFLAGS='-I/root/protobuf/protobuf-2.5.0/src/ -I/root/glog/glog-0.3.3/src/ -I/root/boost/boost-1.53.0/' ./configure --with-mesos=/
 make -j 2
 
 %install
 %make_install
 
+rm -f /usr/local/lib/mesos/*.la
```

Comment:
I don't understand this line and also line # 40. What are these files that we are removing?